src/bosh-director/spec/unit/addon/filter_spec.rb

```diff
@@ -213,7 +238,16 @@ module Addon
 
             it 'applies' do
               expect(addon_include.applies?(nil, nil)).to be(true)
-              expect(addon_include.applies?('anything', instance_group)).to be(true)
+              expect(addon_include.applies?(deployment, instance_group)).to be(true)
+            end
+          end
+
+          describe 'when addon is in deployment manifest and has team filter' do
```

Comment:
Could we add a context here for the `when clause`. Code right now is a bad example but we are going to start to break these whens out.